### DISCUSSION TITLE
Using hummingbird on production ?

### DISCUSSION THREAD

**Discussion by julienrondeau** [PrestaShop/hummingbird#474] [Ideas]
First of all, thanks a lot for this amazing enhanced version of the Classic theme…

I’m a front-end dev, I often use the same dev tools for developing prestashop or Wordpress theme.

because I can’t wait to work with hummingbird… 
my question is : is the theme in version 0.1.5 safe enough to be used for production websites ? 

Or is it better to wait …?

thanks a lot

**Comment by SharakPL** [comment]
I wouldn't recommend it to use in production yet. It still requires bug fixes and improvements. I believe when it's stable enough it will be released with version 1.0.0. Last release was 0.1.5.

Feel free to test it and report any new issue you encounter.
